item-impl/src/main/java/com/example/auction/item/impl/CompletionStageUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.example.auction.item.impl;
+
+import akka.Done;
+import com.datastax.driver.core.BoundStatement;
+import com.lightbend.lagom.javadsl.persistence.cassandra.CassandraReadSide;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+final class CompletionStageUtils {
```

Comment:
These are a bunch of functions that I found useful when writing the read side processor. Maybe some of them should be considered for promotion to the core.